<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1364</width>
    <height>718</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1388</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="posx">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="posy">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="input">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>40</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="point_line_edit">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>100</y>
      <width>351</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>20</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>20</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Number">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="delet">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>40</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deletall">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>520</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="car_width">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>540</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="car_height">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>560</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="showcar">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>550</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>显示车</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_points_image">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>520</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hide_car">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>550</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>隐藏车</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="code_path_edit">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>600</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>D:\\qt\\path1.0\\path\\picture3.png</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>580</y>
      <width>72</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>图片路径</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>520</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>加载输入点</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="length">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>500</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>520</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>轨迹总长：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="number">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>520</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>500</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>轨迹总点数：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plot">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>550</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>速度曲线</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>10</y>
      <width>20</width>
      <height>761</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>490</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>490</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>参数区</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>490</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>-10</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>-10</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>0</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>输入区</string>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>-180</y>
      <width>20</width>
      <height>671</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_11">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>420</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_12">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>420</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>420</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>功能区</string>
    </property>
   </widget>
   <widget class="Line" name="line_13">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>470</y>
      <width>20</width>
      <height>241</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_14">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>0</y>
      <width>20</width>
      <height>141</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>540</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>车子宽度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>600</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>车子长度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>620</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>最大速度：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="max_speed">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>580</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>560</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>减速比：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="kspeed">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>600</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="Line" name="line_16">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>130</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_17">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>0</y>
      <width>20</width>
      <height>141</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_18">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>-10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>0</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>初始化</string>
    </property>
   </widget>
   <widget class="Line" name="line_19">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>-10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="num_btw_two_points">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>40</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;场地图片的水平方向实际长度，单位m&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>竖直长度  ：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;输入的两个控制点之间应该插入多少个贝塞尔点&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>每段点数  ：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_edit_true_height">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>60</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>10.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;场地图片的竖直方向实际长度，单位m&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>水平长度  ：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_edit_true_width">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>100</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>13.3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_trace">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>70</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>加载曲线</string>
    </property>
   </widget>
   <widget class="QPushButton" name="input_next">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>70</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>曲线数+1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_trace_2">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>490</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>加载路径点</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>580</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>最大加速度：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="kspeed_2">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>620</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>25000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sure">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>620</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_points">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>490</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>生成点文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_picture">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>580</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>载入图片</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>140</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>端口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>210</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>140</y>
      <width>93</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>COM1</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="refresh">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>170</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_open_2">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>170</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_baudrate_2">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>210</y>
      <width>93</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
   </widget>
   <widget class="QTextBrowser" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>340</y>
      <width>201</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_send">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>230</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>230</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_send_newline">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>310</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>发送新行</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>310</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>清空接收</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_send_2">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>270</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send_2">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>270</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_send_1">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>250</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send_1">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>250</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_send_3">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>290</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send_3">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>290</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_address">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>460</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>D:\\robocon\\trace.txt</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>440</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>点文件路径</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1364</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
